code-challenges.js

```diff
@@ -47,3 +113,44 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+// I will need to first create a function that determines which variable is larger. I will need to create another function that finds which variable is larger. I will then use console.log and have the second function subtracted from the first function.
+
+const findBig = (num1, num2) => {
+    if (num1 > num2){
+        return num1
+    } else {
+        return num2
+    }
+}
+const bigNum1 = findBig(number1, number2)
+    // finding the larger number in set 1
+const bigNum2 = findBig(number3, number4)
+    // finding the larger number in set 2
+
+// console.log(bigNum1) // checking that my function found the larger number
+
+const findSmall = (num1, num2) => {
+    if (num1 < num2){
+        return num1
+    } else {
+        return num2
+    }
+}
+const smallNum1 = findSmall(number1, number2)
+    // finding the smaller number in set 1
+const smallNum2 = findSmall(number3, number4)
+    // finding the smaller number in set 2
+
+// // console.log(smallNum1)  // checking that it found the smaller number
+
+console.log(bigNum1 - smallNum1)
+    // bigger number of set 1 - smaller number of set 2
+console.log(bigNum2 - smallNum2)
+    // bigger number of set 1 - smaller number of set 2
+
+// Note to assessor: it wasn't until I wrote out this big long answer that I realized I probably could have done this in a single algorthim, but I couldn't get it to work and I know this method works.
+
+// Output:
+// 42
+// 3
```

Comment:
Yea so each of these problems are going to be looking for single algorithms, correct. If we need to create a singular contained piece of code, that makes decisions about which algorithm to use we are looking at the tool called conditionals.